cypress/support/commands.js

```diff
@@ -296,7 +347,79 @@ Cypress.Commands.add('removeTeamMember', (teamURL, username) => {
     cy.visit(`/${teamURL}`);
     cy.get('#sidebarHeaderDropdownButton').click();
     cy.get('#manageMembers').click();
-    cy.focused().type(username, {force: true});
-    cy.get('#removeFromTeam').click({force: true});
+    cy.focused().type(username, {
+        force: true
+    });
+    cy.get('#removeFromTeam').click({
+        force: true
+    });
```

Comment:
Please revert other unnecessary changes above and remain it as is. So we can track it or `git blame` the exact commit as necessary in the future.  It might be due to your `prettier` in case you have that installed.

Reply:
submitted changes!